types/react-form/v2/index.d.ts

```diff
@@ -157,7 +157,7 @@ export interface RadioGroupContext {
 
 export class RadioGroup
     extends React.Component<
-        FieldProps & { children?: ((props: FieldApi) => RenderReturn) | RenderReturn | undefined }
+        Omit<FieldProps, 'children'> & { children?: ((props: FieldApi) => RenderReturn) | RenderReturn | undefined }
```

Comment:
Previous intersection only worked because we intersected `{} & (props) => RenderReturn`.